app/forms/declaration/ContainerYesNo.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration
+import forms.DeclarationPage
+import forms.Mapping.requiredRadio
+import forms.declaration.ContainerAdd.maxContainerIdLength
+import play.api.data.Forms.text
+import play.api.data.{Form, Forms}
+import play.api.libs.json.Json
+import uk.gov.voa.play.form.ConditionalMappings.mandatoryIfEqual
+import utils.validators.forms.FieldValidator.{isAlphanumeric, noLongerThan, nonEmpty}
+
+case class ContainerYesNo(id: Option[String])
```

Comment:
I am not sure about this name. Name suggest `Boolean` but it contains id.

Reply:
Renamed to 'ContainerFirst'.  It's now better aligned with the view that uses it.